app/views/users/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<%# Regular Form %>
+
+<!--
+<form action="/users" method="post" accept-charset="UTF-8">
```

Comment:
Kindly use `users_path` from Rails helpers instead of `/users`

Reply:
Hi, thank you for your review!